DigitalLearningSolutions.Web/Views/TrackingSystem/DelegateApprovals/Index.cshtml

```diff
@@ -0,0 +1,38 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem
+@model DelegateApprovalsViewModel
+
+@{
+  ViewData["Title"] = "Approve delegate registrations";
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <h1 class="nhsuk-heading-xl">Approve delegate registrations</h1>
+    @if (Model.Delegates.Any()) {
+      @foreach(var delegateApproval in Model.Delegates)
+      {
+        <partial name="_delegateExpandable" model="delegateApproval" />
+      }
+      <div class="nhsuk-grid-row">
+        <div class="nhsuk-grid-column-full">
+          <a class="nhsuk-button" href="" role="button">
+            Approve All
+          </a>
+        </div>
+      </div>
+    } else {
+      <div class="nhsuk-grid-row">
+        <div class="nhsuk-grid-column-full">
+          <p>No accounts need approval.</p>
+        </div>
+      </div>
+      <div class="nhsuk-grid-row">
+        <div class="nhsuk-grid-column-full">
+          <a class="nhsuk-button" href="" role="button">
+            Manage approved delegates
+          </a>
+        </div>
+      </div>

```

Comment:
Could this be an NHS action link (we should have a VC for it) https://service-manual.nhs.uk/design-system/components/action-link

Can we also link to [old url] + /tracking/delegates?